hack/local-up-cluster.sh

```diff
@@ -897,6 +918,8 @@ if [[ "${START_MODE}" != "kubeletonly" ]]; then
   start_apiserver
```

Comment:
and that should be done after creating the admin kubeconfig, and should use the admin kubeconfig